src/site/content/en/payments/handling-optional-payment-information/index.md

```diff
@@ -200,13 +200,271 @@ object.
 Use the object to update the UI on the frontend. See [Reflect the updated
 payment details](#reflect-the-updated-payment-details).
 
+## Inform the merchant of a shipping address change {: #shipping-address-changes }
+
+Payment apps can provide the customer's shipping address to the merchant as part
+of the result of a payment transaction.
```

Comment:
I think we can rewrite this as "Payment apps can provide a customer's shipping address to the merchant as part of a payment transaction."